### PR TITLE
code consistency in addon

### PR DESCRIPTION
**Pull request by sbatson5**
Hey all!

Love this addon--using it on a client project. Was debugging some drag event stuff and hopped into the code and saw some inconsistency around function declarations and `var` versus `let`.  This PR doesn't change any functionality, but at least gets a few of these files to be consistently styled.

### CODE REVIEW

File: addon/components/draggable-object-target.js [hunk 1]
```diff
@@ -6,31 +6,33 @@ export default Ember.Component.extend(Droppable, {
...
   },
 
-  acceptDrop: function(event) {
+  acceptDrop(event) {
     this.handleDrop(event);
     //Firefox is navigating to a url on drop sometimes, this prevents that from happening
     event.preventDefault();
   },
-  handleDragOver: function(event) {
+
```
Comment (sbatson5) [inline addon/components/draggable-object-target.js]: The other functions seemed to have line breaks between them, so I added these.

File: addon/components/draggable-object.js [hunk 2]
```diff
@@ -2,7 +2,6 @@ import Ember from 'ember';
 
 export default Ember.Component.extend({
   dragCoordinator: Ember.inject.service(),
-  tagName: "div",
```
Comment (sbatson5) [inline addon/components/draggable-object.js]: Components are `div` by default, so there's no need to declare it here. Let me know if there was reasoning behind this.

File: addon/components/draggable-object.js [hunk 3]
```diff
@@ -31,26 +25,27 @@ export default Ember.Component.extend({
   },
 
   didInsertElement() {
-    Ember.run.scheduleOnce('afterRender', ()=> {
-      let self = this;
```
Comment (sbatson5) [inline addon/components/draggable-object.js]: This addon seems to use arrow functions throughout. Using them here should remove the need for assigning `self`

File: addon/components/draggable-object.js [hunk 4]
```diff
@@ -80,7 +75,7 @@ export default Ember.Component.extend({
       event.preventDefault();
       return;
     } else {
-      Ember.run.later(()=> {
+      Ember.run.next(()=> {
```
Comment (sbatson5) [inline addon/components/draggable-object.js]: `run.later` expects 2 params: the callback and the time delay.  Without setting a time for the function to wait, it is essentially a `next`.

**Comment by dgavey** [comment]
This is great thanks for the code style updates.  I would love to rewrite this addon, it's still on my (growing) todo list.

**Pull request merged**